client/src/lib/constants.ts

```diff
@@ -5,9 +5,11 @@ export const SOL_DID_PROGRAM_ID = new PublicKey(
   'idDa4XeCjVwKcprVAo812coUQbovSZ4kDGJf2sPaBnM'
 );
 
-export const DOA_PROGRAM_ID = new PublicKey(
-  'CFE6uDuLK7Hd9HVQqWkXU3nTmdZUm5KCEtFcEfbSTZLy'
-);
+export const CRYPTID_PROGRAM_ID =
+  process.env.STAGE === 'dev'
+    ? new PublicKey('tcrjc6mfci8bQLmXPfrVw9fJm9Y5tr268tByjSQmSe9')
+    : new PublicKey('crypt1GWL27FYSg7gEfJVzbc8KzEcTToFNmaXi9ropg');
```

Comment:
ok, so the `default` is `crypt1GWL27FYSg7gEfJVzbc8KzEcTToFNmaXi9ropg`, e.g. when the env variable is not set. I think that is the way to go. (versus making the default `tcrjc6mfci8bQLmXPfrVw9fJm9Y5tr268tByjSQmSe9`)